pages/ecosystem.vue

```diff
@@ -301,6 +304,17 @@ const filteredMembersSorted = computed<Member[]>(() => {
 
   return filteredMembers.value.sort((a, b) => a.name.localeCompare(b.name));
 });
+
+function getTierProjectCount(tierName: string) {
```

Comment:
To increase type security, I think that we should probably have a type `TierName` exported from the **types/ecosystem.ts** file.

We can also reuse it in that file in the `Member` interface and in the `Tier` interface, and probably in some other places as well.

Reply:
Addressed [here](https://github.com/Qiskit/qiskit.org/pull/3477/commits/bab37d63bbe1aef52efa3d89c1f35d1e544cc5f8)